tutorials/shading/your_first_shader/your_first_canvasitem_shader.rst

```diff
@@ -0,0 +1,212 @@
+.. _doc_your_first_canvasitem_shader:
+
+Your first CanvasItem shader
+============================
+
+Introduction
+------------
+
+Shaders are special programs that execute on the GPU and are used for rendering 
+graphics. All modern rendering is done with shaders. For a more detailed description
+of what shaders are please see :ref:`What are shaders <doc_what_are_shaders>`.
+
+This tutorial will focus on the practical aspects of writing shader programs by walking
+readers through the process of writing a shader with both vertex and fragment functions.
```

Comment:
In the first doc you're using first person ("This tutorial series will walk you through writing your first shader."), but in this one you're much more formal and third person ("This tutorial will ... walking readers through the process"). I prefer the former, but either way we should be consistent.